DOCS-696/711/723/722/480: align component model pages and example json templates (#1170)
viamrobotics/docs: docs/components/input-controller/mux.md

Click on the **Components** subtab and navigate to the **Create component** menu.
Enter a name for your input controller, select the type `input_controller`, and select the `mux` model.

Click **Create component**.

![An example configuration for a multiplexed input controller component in the Viam App config builder](../img/mux-input-controller-ui-config.png)

Edit and fill in the attributes as applicable.

{{% /tab %}}
{{% tab name="JSON Template" %}}

```json {class="line-numbers linkable-line-numbers"}
{
  "components": [
    {
      "name": "<your-mux-input-controller-name>",
      "type": "input_controller",
      "model": "mux",
      "attributes": {
        "sources": [

>           "<your-gamepad-input-controlle-name",

Reviewer: ```suggestion
          "<your-gamepad-input-controller-name",
```

The following attributes are available for `mux` input controllers:

| Name | Type | Inclusion | Description |
| ---- | ---- | --------- | ----------- |

> | `sources` | array | **Required** | The `name` of each controller you wish to combine. |

Reviewer: ```suggestion
| `sources` | array | **Required** | The `name`s of all controller components you wish to combine. |
```
Or something to make it clearly plural, and I think leaving "component" in helps them know it's the config file names?

Author: Good suggestion but I think this should be fine in the more brief form w/ example and "each"

Author: Alternative: 
```suggestion
| `sources` | array | **Required** | The `name` of each controller component you wish to combine the input from. |
```
